docs/tutorials/adv_tutorial.ipynb

```diff
@@ -15,6 +15,8 @@
     "\n",
     "So far, we've built a model, run it, and analyzed some output afterwards. However, one of the advantages of agent-based models is that we can often watch them run step by step, potentially spotting unexpected patterns, behaviors or bugs, or developing new intuitions, hypotheses, or insights. Other times, watching a model run can explain it to an unfamiliar audience better than static explanations. Like many ABM frameworks, Mesa allows you to create an interactive visualization of the model. In this section we'll walk through creating a visualization using built-in components, and (for advanced users) how to create a new visualization element.\n",
     "\n",
+    "**Note for Jupyter users: Due to conflicts with the tornado server Mesa uses and Jupyter, the interactive browser of your model will load but likely not work. This will require you to use run the code from .py files. The Mesa development team is working to develop a** [Jupyter compatible interface](https://github.com/projectmesa/mesa/issues/1363>).**\n",
```

Comment:
Does the link work with the `>` at the end?

Reply:
I tried. It didn't work.

Reply:
Oh geez, thanks for the catch... that epitomizes the pain of having an  `.rst` and `.ipynb`. Fixed